test/upgrade/test-upgrade.bats

```diff
@@ -10,88 +10,88 @@ load helpers
 # options both in the container podman and out here: that's the only
```

Comment:
yikes, most of this file seems to have been modified by my IDE, I'll check to see that i can disable that on save to simplify the changes.